Referenced by issue #59069, Python: Bmesh "from_object" broken in 2.8
